package.json

```diff
@@ -60,6 +64,11 @@
     "react-addons-test-utils": "^0.14.2",
     "react-dom": "^0.14.2",
     "rimraf": "^2.4.2",
+    "rollup": "^0.21.0",
```

Comment:
Interesting choice :). Webpack is the standard but Rollup does some really neat stuff. The future is now!